tests/hidden-posts-testcase.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
No need for a custom TestCase unless there are actually some tests.

Also, creating a global like this isn't necessarily the best way of grabbing the class under test. For instance, there is no `$hidden_posts` global defined in the main code when the only class is [instantiated](https://github.com/Automattic/hidden-posts/blob/209e6026f1f8ee9c1e4d18c6403ed30eaef8d572/hidden-posts.php#L192), so the logic in this setUp() isn't going to work.